mapper/javabean/src/main/java/org/hibernate/search/mapper/javabean/massindexing/loader/JavaBeanIndexingOptions.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.javabean.massindexing.loader;
+
+import org.hibernate.search.mapper.pojo.massindexing.loader.MassIndexingOptions;
+import org.hibernate.search.mapper.javabean.transaction.TransactionOptions;
+
+public interface JavaBeanIndexingOptions extends MassIndexingOptions, TransactionOptions {
```

Comment:
API in this module don't use the `JavaBean` prefix.

Also, it's not necessary to extend a common superinterface `MassIndexingOptions` here and in the Hibernate ORM module. We generally try to avoid common APIs in the two mappers (even though you already did that elsewhere...).

Please rename this interface to `MassIndexingOptions`, and move all relevant methods from `org.hibernate.search.mapper.pojo.massindexing.loader.MassIndexingOptions` to here.

Reply:
Some functions are used in mapper -ojeo. Some will always be the same for mapper-javabean and mapper-orm.